### ISSUE TITLE
Invalid user exception

### ISSUE THREAD

**Issue by kstan79** [smi2/phpClickHouse#88]
I'm tried this library, it work fine when we submit correct during connect database, it work well. if we submit invalid user, php will failed. Hope it can use others handling like return false or etc to prevent php exit.

**Comment by 2ur1st** [comment]
To throw exception it is right way to process incorrect behavior, use should use `try - catch` for your project like this
```
$config = [
    'host' => 'localhost',
    'port' => '8123',
    'username' => 'invalid_user',
    'password' => ''
];
try {
    $db = new ClickHouseDB\Client($config);
    $db->database('default');
} catch (InvalidArgumentException $e) {
     // you can log this error message 
     return false;
}
```

**Comment by kstan79** [comment]
thanks reply, i did this before, php failed.

**Comment by 2ur1st** [comment]
can you show a stacktrace of your code ?

**Comment by kstan79** [comment]
let me try again and come back to you

**Comment by isublimity** [comment]
If need reopen or new issue
